src/Workspaces/Core/Portable/Diagnostics/InternalDiagnosticsOptions.cs

```diff
@@ -21,5 +21,8 @@ internal static class InternalDiagnosticsOptions
 
         public static readonly Option2<DiagnosticMode> RazorDiagnosticMode = new(nameof(InternalDiagnosticsOptions), nameof(RazorDiagnosticMode), defaultValue: DiagnosticMode.Pull,
             storageLocation: new LocalUserProfileStorageLocation(LocalRegistryPath + "RazorDiagnosticMode"));
```

Comment:
why do we need this option anymore.  it can't be flipped by users.  that actually makes thsi somewhat bad.  someone coudl have it flipped to "push", then we removed the UI option, and now they're stuck in this mode (unless they literally edited the registry).